src/api/compatibility/application/CompatibleApplicationResourceTreeItem.ts

```diff
@@ -69,7 +69,7 @@ export class CompatibleResolvedApplicationResourceTreeItem extends AzExtParentTr
                 subscription: __subscription,
                 parent: undefined,
                 removeChildFromCache: () => {
-                    //
+                    this.treeDataProvider.refreshUIOnly(undefined);
```

Comment:
It's for compat so it matters less, but do we have a handle to the child's parent, to scope the refresh?

Reply:
Follow up: I've had that scoped refresh thing in a git stash, will make a PR for it.